src/Cardano/Wallet/Api/Types.hs

```diff
@@ -176,13 +180,21 @@ instance ToJSON  WalletPostData where
 
 instance FromJSON (ApiT (Passphrase "encryption")) where
     parseJSON = parseJSON >=> \case
-        t | T.length t < 10 ->
-            fail "passphrase is too short: expected at least 10 chars"
-        t | T.length t > 255 ->
-            fail "passphrase is too long: expect at most 255 chars"
+        t | T.length t < passphraseMinLength ->
+            fail $ "passphrase is too short: expected at least "
+                <> show passphraseMinLength <> " chars"
+        t | T.length t > passphraseMaxLength ->
+            fail $ "passphrase is too long: expected at most "
+                <> show passphraseMaxLength <> " chars"
         t ->
             return $ ApiT $ Passphrase $ BA.convert $ T.encodeUtf8 t
 
+passphraseMinLength :: Int
+passphraseMinLength = 10
+
+passphraseMaxLength :: Int
+passphraseMaxLength = 255
```

Comment:
Perhaps these named constants should be located in the `AddressDerivation` module?

Reply:
Not really because, that's really just a user-facing thing here. From the derivation perspective, we don't care about it and the passphrase could as well be empty. Actually, I've discussed this with Charles Morgan and, in the short-long run, we do want to enforce some better strength validation at the API boundary. Yet, this doesn't have to "leak" through the rest IMO.